src/LibraryUtilities.ts

```diff
@@ -295,3 +297,24 @@ export function searchItemResursive(items: ItemData[], text: string) {
         search(text, item);
     }
 }
+
+export function getHighlightedText(text: string, highlightedText: string) {
+    if (highlightedText.length == 0) {
+        return text;
+    }
+
+    var regex = new RegExp('' + highlightedText + '', 'gi');
```

Comment:
Oh at first I added slash in the quotation marks, later I found that it's not needed. I'll remove them.